web_server.py

```diff
@@ -0,0 +1,93 @@
+
+"""
+This benchmark runs a trivial Twisted Web server and client and makes as many
+requests as it can in a fixed period of time.
+
+A significant problem with this benchmark is the lack of persistent connections
+in the HTTP client.  Lots of TCP connections means lots of overhead in the
+kernel that's not really what we're trying to benchmark.  Plus lots of sockets
+end up in TIME_WAIT which has a (briefly) persistent effect on system-wide
+performance and makes consecutive runs of the benchmark vary wildly in their
+results.
+"""
+
+from twisted.internet.protocol import Factory
+from twisted.internet.endpoints import TCP4ClientEndpoint
+from twisted.internet.defer import Deferred
+from twisted.web.server import Site
+from twisted.web.static import Data
+from twisted.web.resource import Resource
+from twisted.test.proto_helpers import AccumulatingProtocol
+
+from benchlib import Client, driver
+
+
+class Client(Client):
```

Comment:
A web server benchmark doesn't really fit well with the `Client` helper... I guess you've made something work here but in the long term it would probably be better to build a `Server` benchmark helper.